extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -125,13 +124,9 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
       const lookupResult = await this.view.pubLookup.lookupEmail(email);
       if (lookupResult && email) {
         if (lookupResult.pubkey) {
-          const parsed = await opgp.key.readArmored(lookupResult.pubkey);
-          const key = parsed.keys[0];
-          if (!key) {
-            console.info('Dropping found but incompatible public key', { for: email, err: parsed.err ? ' * ' + parsed.err.join('\n * ') : undefined });
-            lookupResult.pubkey = null; // tslint:disable-line:no-null-keyword
-          } else if (! await PgpKey.usableForEncryption(lookupResult.pubkey) && ! await PgpKey.expired(key)) { // Not to skip expired keys
-            console.info('Dropping found+parsed key because getEncryptionKeyPacket===null', { for: email, fingerprint: await PgpKey.fingerprint(parsed.keys[0]) });
+          const key = lookupResult.pubkey;
+          if (!key.usableForEncryption && !PgpKey.expired(key)) { // Not to skip expired keys
+            console.info('Dropping found+parsed key because getEncryptionKeyPacket===null', { for: email, fingerprint: key.id });
```

Comment:
It may be good to propagate this info to the user as a modal or similar? (make another issue)